### PR TITLE
DS-3061: Use workflow.framework instead of workflow.workflow.framework

### PR DESCRIPTION
**Pull request by pnbecker**
https://jira.duraspace.org/browse/DS-3061

### CODE REVIEW

**Comment by pnbecker** [comment]
I tested this roughly: On JSPUI and XMLUI I submited a document without any problems using the original or basic workflow, without testing any workflow steps.

I'm totally sure the configurable workflow won't work without this PR. Nevertheless I couldn't tested it as I was unable to activate it neither with nor without this PR.

File: dspace-api/src/main/java/org/dspace/submit/step/CompleteStep.java [hunk 1]
```diff
@@ -87,7 +87,8 @@ public int doProcessing(Context context, HttpServletRequest request,
         boolean success = false;
         try
         {
-            if(ConfigurationManager.getProperty("workflow","workflow.framework").equals("xmlworkflow")){
+            if(StringUtils.equals(configurationService.getProperty("workflow.framework"), "xmlworkflow"))
+            {
```
Comment (tdonohue) [inline dspace-api/src/main/java/org/dspace/submit/step/CompleteStep.java]: As mentioned in IRC, this change will conflict with the one in #1280 .  I think the fix in #1280 is the correct one (just for this class).

**Comment by pnbecker** [comment]
I cherry-picked @KevinVdV solution from #1280 and merged it with my one.

**Comment by tdonohue** [comment]
+1 via code review. This looks like obvious fixes to me, as it's mostly just replacing `ConfigurationManager` calls with corresponding `ConfigurationService` calls in order to ensure we are not doubly prefixing the configuration.

**Comment by tdonohue** [comment]
I'm merging this as these configuration issues are also getting in the way of testing other submission-based PRs, e.g. #1162

**Pull request merged**